olad/www/new/index.html

```diff
@@ -54,6 +54,15 @@
  </div>
  <!-- /.container-fluid -->
 </nav>
+<noscript>
+  <br>
+  <p align="center">
+    <strong>This interface requires JavaScript to function. If you
```

Comment:
That is how Emacs indents HTML. To change that I could put the `strong` tags on their own lines, would you prefer that?